src/components/SiteItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const site = props => (
+  <div onClick={props.clicked}>{props.site.properties.site_name} ( click me! )</div>
+);
+
```

Comment:
Okay, this could totally use some help with PropTypes for maintainability! 

```js
import PropTypes from 'prop-types'

...

site.propTypes = {
  clicked = PropTypes.func.isRequired,
  site: PropTypes.shape({
    properties: PropTypes.shape({
      site_name: PropTypes.string.isRequired,
    })
  })
}
```